src/client/components/battlekards_details.vue

```diff
@@ -45,10 +45,17 @@
         <button
           v-if="isMyTurn && selectedCard.cardField === 'myHand' && !myPlayerHasSummoned"
           id="summonButton"
-          v-on:click="summon"
+          v-on:click="summon('attak')"
         >
           Summon Attak
         </button>
+        <button
+          v-if="isMyTurn && selectedCard.cardField === 'myHand' && !myPlayerHasSummoned"
```

Comment:
Adding an extra `v-if` here is unnecessary and could mess with the following `v-else-if`s. You can just wrap both buttons with:

```html
<div v-if="isMyTurn && selectedCard.cardField === 'myHand' && !myPlayerHasSummoned">
  <!-- buttons here -->
</div>
```